.github/workflows/CI.yml

```diff
@@ -11,19 +11,17 @@ jobs:
 
     strategy:
       matrix:
-        node: [12, 14, 16]
-        os: [ubuntu-latest, macOS-latest, windows-latest]
+        node: [16, 18, 20]
```

Comment:
Officially we only support Node LTS, so this can just be `[20, 22]` instead

Does the Mac issue exist in those versions? We should keep it active, it's best to resolve this issue instead of removing support 😅

>

Reply:
Upgraded bunch of dependencies and it works now.